modules/s2/redshift/doc/ST_ASID_POLYFILL_BBOX.md

```diff
@@ -0,0 +1,42 @@
+### ST_ASID_POLYFILL_BBOX
+
+{{% bannerNote type="code" %}}
+s2.ST_ASID_POLYFILL_BBOX(id, resolution)
+{{%/ bannerNote %}}
+
+**Description**
+
+Returns a SUPER containing an array of s2 cell IDs that cover a planar bounding box. Note that this is
+a compact coverage (polyfill), so the bounding box is covered with the least amount of cells 
+by using the largest cells possible.
+
+Two optional arguments can be passed with the minimum and maximum resolution for cogerage. If you desire a
+coverage at a single resolution level, simply set the maximum and minimum longitude to be of equal value.
+
+* `min_latitude`: `FLOAT` minimum latitude of the bounding box.
```

Comment:
Its better to just use FLOAT. I started using NUMERIC because I though RS wouldn't automatically cast integers to float but it does so no need for them anymore. Updated in `sql/LONGLAT_ASID.sql`